crypto/crypto.go

```diff
@@ -43,16 +44,22 @@ func (f SignatureAlgorithm) String() string {
 	return [...]string{"UNKNOWN", "BLS_BLS12381", "ECDSA_P256", "ECDSA_secp256k1"}[f]
 }
 
-// MinSeedLength returns the minimum seed length that can safely be used to generate private keys with this algorithm.
-func (f SignatureAlgorithm) MinSeedLength() int {
+// MinSeedLength is a generic minimum seed length to guarantee a minimum entropy.
+// It is used when the seed source is not necessary a CSPRG and the seed
+// should be expanded before being passed to the key generation process.
+const MinSeedLength = crypto.MinSeedLen
+
+// minSeedLength returns the minimum seed length that can safely be used to generate private keys with this algorithm.
```

Comment:
I think we might not need this function as it's only used in one place. Feel free to remove it or not